moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -0,0 +1,303 @@
+from pathlib import Path
+from typing import Optional, List
+import logging
+from ament_index_python.packages import get_package_share_directory
+
+from parameter_builder import ParameterBuilder, load_yaml, load_xacro
+
+
+class MoveItConfigs(object):
+    __slots__ = [
+        "__robot_description",
+        "__robot_description_semantic",
+        "__robot_description_planning",
+        "__robot_description_kinematics",
+        "__planning_pipelines",
+        "__trajectory_execution",
+        "__planning_scene_monitor",
+        "__move_group_capabilities",
+        "__move_group",
+        "__joint_limits",
+        "__moveit_cpp",
+        "__cartesian_limits",
+    ]
+
+    def __init__(self):
+        self.robot_description = {}
+        self.robot_description_semantic = {}
+        self.robot_description_planning = {}
+        self.robot_description_kinematics = {}
+        self.planning_pipelines = {}
+        self.trajectory_execution = {}
+        self.planning_scene_monitor = {}
+        self.move_group_capabilities = {}
+        self.move_group = {}
+        self.joint_limits = {}
+        self.moveit_cpp = {}
+        self.cartesian_limits = {}
+
+    @property
+    def robot_description(self):
+        return self.__robot_description
```

Comment:
[PEP-8](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables) seems to suggest using double underscore prefixes for simple member variables is not recommended.

Reply:
Ping to fix this